package.json

```diff
@@ -69,13 +72,13 @@
   "devDependencies": {
     "bump-version": "^0.5.0",
     "chai": "~1.7.2",
+    "eslint": "^8.45.0",
     "mocha": "^8.2.1",
     "proxyquire": "^2.1.3",
     "sinon": "^9.0.3",
     "snyk": "^1.381.1"
   },
-  "optionalDependencies": {},
   "engines": {
-    "node": "^12.19"
+    "node": "^18.16.0"
```

Comment:
node18

Reply:
I feel like I've noted this before, but I think still allowing older versions would allow for an easier update path. Does this work with 12 / 16, etc? If so, maybe leaving this line alone for the time being would avoid a major version.